### ISSUE TITLE
Match case for class items support

### ISSUE THREAD

**Issue by mastrolube** [dashingsoft/pyarmor#1756]
Hello, I've refactored a little bit my code and found out that pyarmor is not supporting `match case` with class items. (I'm on pro version)

That's the error: 

> patterns may only match literals and attribute lookups

```
from enum import Enum, auto

class ExampleEnum(Enum):
    OPTION_A = auto()
    OPTION_B = auto()
    OPTION_C = auto()

def example_function(value: ExampleEnum):
    match value:
        case ExampleEnum.OPTION_A:
            # perform action
        case ExampleEnum.OPTION_B:
            # perform action
        case ExampleEnum.OPTION_C:
            # perform action
        case _:
            # default action
```
What's the reason for that? Will that be supported in the future?
Thanks, Dennis

**Comment by jondy** [comment]
Can't be reproduced. What I got by Pyarmor 8.5.7 with Python 3.10 in Darwin:
```python
...
class pyarmor__1(pyarmor__6):
    pyarmor__2 = pyarmor__7()
    pyarmor__3 = pyarmor__7()
    pyarmor__4 = pyarmor__7()

def pyarmor__5(value: pyarmor__1):
    match value:
        case pyarmor__1.pyarmor__2:
            pass
        case pyarmor__1.pyarmor__3:
            pass
        case pyarmor__1.pyarmor__4:
            pass
        case _:
            pass
```

Try to clean local configuration path `mv .pyarmor .pyarmor.bak` and test it again.

**Comment by mastrolube** [comment]
Hello!
I found out that it's related to: `--obf-code 2`

 `pyarmor -d gen -O dist test.py --obf-code 2` where test.py contains the code above
 

> INFO     obfuscating file test.py
> DEBUG    process inline marker
> DEBUG    parse script
> DEBUG    process co
> DEBUG    process attribute
> DEBUG    unknown error, please check pyarmor.error.log
> ERROR    patterns may only match literals and attribute lookups
> ERROR    somthing is wrong

Also there is a typo in `somEthing` :)

Thanks!
Dennis

**Comment by jondy** [comment]
Got it. Thanks, all of them will be fixed in next release

**Comment by jondy** [comment]
Fixed in 8.5.8
